docs/src/development/proposals/MEP11/README.md

```diff
@@ -0,0 +1,79 @@
+# Auditing of metal-stack resources
+
+Currently no logs of the ownership of resources like machines, networks, ips and volumes are generated or kept.Though due to legal requirements data centers are required to keep track of this ownership over time to prevent liability issues when opening the platform for external users.
+
+In this proposal we want to introduce a flexible and low-maintenance approach for auditing on top of [Meilisearch](https://www.meilisearch.com/).
+
+## Overview
+
+In general our auditing logs will be collected by a request interceptor or middleware. Every request and response will be processed and eventually logged to Meilisearch.
+Meilisearch will be configured to regularly create chunks of the auditing logs. These finished chunks will be backed up to a S3 compatible storage with a read-only option enabled.
+
+Of course sensitve data like session keys or passwords will be redacted before logging. We want to track relevant requests and responses. If auditing the request fails, the request itself will be aborted and will not be processed further. The requests and responses that will be audited will be annotated with a correlation id.
+
+Transferring the meilisearch auditing data chunks to the S3 compatible storage will be done by a cronjob that is executed periodically.
```

Comment:
Would be nice to provide a technical idea of how to do this. Also maybe one word that the logs are deleted automatically as well because they are probably not allowed to be kept forever?

Reply:
Good points. Can you have another look?
I also added a few links and described the missing pieces.